ingestion/src/metadata/ingestion/stage/migrate.py

```diff
@@ -0,0 +1,254 @@
+    report: StageStatus
+
+    def __init__(
+        self,
+        config: FileSinkConfig,
+        metadata_config: OpenMetadataConnection,
+    ):
+        self.config = config
+        self.metadata_config = metadata_config
+        self.report = StageStatus()
+        self.table_file = open(f"{self.config.filename}/table.json", "w")
+        self.user_file = open(f"{self.config.filename}/user.json", "w")
+        self.team_file = open(f"{self.config.filename}/team.json", "w")
+        self.topic_file = open(f"{self.config.filename}/topic.json", "w")
+        self.pipeline_file = open(f"{self.config.filename}/pipeline.json", "w")
+        self.glossary_file = open(f"{self.config.filename}/glossary.json", "w")
+        self.glossary_term_file = open(
+            f"{self.config.filename}/glossary_term.json", "w"
+        )
+        self.tag_file = open(f"{self.config.filename}/tag.json", "w")
+        self.database_service_file = open(
+            f"{self.config.filename}/database_service.json", "w"
+        )
+        self.role_file = open(f"{self.config.filename}/role.json", "w")
+        self.policy_file = open(f"{self.config.filename}/policy.json", "w")
+        self.pipeline_service_file = open(
+            f"{self.config.filename}/pipeline_service.json", "w"
+        )
+        self.messaging_service_file = open(
+            f"{self.config.filename}/messaging_service.json", "w"
+        )
+        self.table_file.write("[\n")
+        self.user_file.write("[\n")
+        self.tag_file.write("[\n")
+        self.team_file.write("[\n")
+        self.topic_file.write("[\n")
+        self.pipeline_file.write("[\n")
+        self.glossary_file.write("[\n")
+        self.glossary_term_file.write("[\n")
+        self.role_file.write("[\n")
+        self.policy_file.write("[\n")
+        self.database_service_file.write("[\n")
+        self.messaging_service_file.write("[\n")
+        self.pipeline_service_file.write("[\n")
+        self.wrote_table = False
+        self.wrote_user = False
+        self.wrote_team = False
+        self.wrote_topic = False
+        self.wrote_dashboard = False
+        self.wrote_pipeline = False
+        self.wrote_glossary = False
+        self.wrote_glossary_term = False
+        self.wrote_location = False
+        self.wrote_chart = False
+        self.wrote_database_service = False
+        self.wrote_role = False
+        self.wrote_policy = False
+        self.wrote_tag = False
+        self.wrote_messaging_service = False
+        self.wrote_pipeline_service = False
+
+        self.metadata = OpenMetadata(
+            OpenMetadataConnection.parse_obj(self.metadata_config)
+        )
+
+    @classmethod
+    def create(cls, config_dict: dict, metadata_config: OpenMetadataConnection):
+        config = FileSinkConfig.parse_obj(config_dict)
+        return cls(config, metadata_config)
+
+    def stage_record(self, record: Entity) -> None:
+        if isinstance(record, Table):
```

Comment:
This can indeed be a singledispatch

Reply:
We can use this `stage` as a chance to try out a new structure and clean the rest of the ingestion later on